cpp/oneapi/dal/detail/ccl/communicator.hpp

```diff
@@ -325,6 +325,14 @@ class ccl_communicator_impl : public ccl_interface_selector<MemoryAccessKind>::t
         return default_root_;
     }
 
+    bool get_mpi_gpu_support() override {
```

Comment:
may be it makes sense to rename it and add smth related to offloading, due to its little bit confusing